internal/pkg/auth/auth_token_handler.go

```diff
@@ -165,13 +160,9 @@ func (a *AuthTokenHandlerImpl) checkSSOEmailMatchesLogin(client *ccloudv1.Client
 }
 
 func login(client *ccloudv1.Client, req *ccloudv1.AuthenticateRequest) (*ccloudv1.AuthenticateReply, error) {
-	if isOkta(client) {
+	if sso.IsOkta(client.BaseURL) {
```

Comment:
GetIsSso vs. IsOkta -- naming conventions / consistency, wdyt?

Reply:
`GetIsSso()` is autogenerated by the SDK, so we don't have much control over it!

Reply:
It's a panic-safe function for the `IsSso` field.